uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/GroupSeparatorItem.php

```diff
@@ -14,6 +14,6 @@ class CRM_HRContactActionsMenu_Component_GroupSeparatorItem implements ActionsGr
    * {@inheritDoc}
    */
   public function render() {
-    return '<hr>';
+    return '<hr class="crm_contact-actions--dark-gray-blue">';
```

Comment:
what's this class for? Please add screenshot

Reply:
This class makes the `<hr>`  in dark gray blue in color.

Reply:
why do you need an entirely new class for it? Can't you have the parent simply setting the color of the <hr>?